### PR TITLE
Add service map

### PR DESCRIPTION
**Pull request by divya051988**
- Add a service map to be displayed in Neo4j.
-

### CODE REVIEW

**Review by amoriartyCH — APPROVED** [review]

**Review by ch-tstrickland — APPROVED** [review]
Looks good. It would be great if there was a link to the contracts for the integration_outgoing section.

File: service-map.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  "specification": "https://github.com/companieshouse/api.ch.gov.uk-specifications",
+  "public": "false",
+  "users": "internal",
+  "code_analysis_link": " http://code-analysis.platform.aws.chdev.org:9000/",
+  "test_automation_repo": "https://github.com/companieshouse/taf-api-karate",
+  "service_area": "Maintain a Company",
+  "service_component": "Accounts",
+  "language": "Java",
+  "integration_outgoing": [
+    "transactions_accounts,SENDS_DATA_TO",
```
Comment (ch-tstrickland) [inline service-map.json:17]: Quotes look wrong here.

File: service-map.json [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+  "type": "Service",
+  "name": "company-accounts.api.ch.gov.uk",
+  "description": "The Companies House API for handling company accounts",
+  "owned_by": "Aardvark",
+  "calls_via": "eric",
+  "git_repo": "https://github.com/companieshouse/company-accounts.api.ch.gov.uk",
+  "specification": "https://github.com/companieshouse/api.ch.gov.uk-specifications",
+  "public": "false",
+  "users": "internal",
+  "code_analysis_link": " http://code-analysis.platform.aws.chdev.org:9000/",
```
Comment (mreis-ch) [inline service-map.json:11]: Just spotted a space at the beginning of the URL!

**Comment by divya051988** [comment]
> Looks good. It would be great if there was a link to the contracts for the integration_outgoing section.

Good point . These are pre-defined values - Also isn't that the same as specifications ?

**Pull request merged**